training-maquetado/scss/_styles.scss

```diff
@@ -3,6 +3,10 @@
   font-size: $text-small;
 }
 
+.main-container {
+  padding: 90px 0;
```

Comment:
i would put this on a variable like `$navbar-height` and use it in navbar.scss too